### PR TITLE
JIRA review

### PR DESCRIPTION
**Pull request by vanmaegima**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

This includes information for imx8mn, customer feedback and general improvements.

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for fb308be61b1ffceb19de49aa31c2145b67efec7e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1691/docs/artifacts/html/index.html

**Comment by kprosise** [comment]
We can ignore the link-check error, it's due to an external link not being available, which I will be investigating and likely patching.

**Comment by kprosise** [comment]
@vanmaegima, can you provide more information on what this PR addresses? And is it to be merged pending reviews? Thank you in advance and sorry for any trouble.

File: source/reference-manual/security/offline-keys.rst [hunk 1]
```diff
@@ -35,10 +35,15 @@ This only needs to be done **once** to gather the root keys offline. It
 will gather the keys offline and make a singular, first, rotation at
 the same time.
 
-This can be done in fioctl with::
+.. note:: Factories owners that received the TUF keys tarball via email at
```
Comment (kprosise) [inline source/reference-manual/security/offline-keys.rst:38]:
```suggestion
.. note:: Factory owners that received the TUF keys tarball via email at
```

File: source/reference-manual/security/offline-keys.rst [hunk 2]
```diff
@@ -35,10 +35,15 @@ This only needs to be done **once** to gather the root keys offline. It
...
+    Factory creation (former way) and have not downloaded/rotated the initial
+    keys can proceed with the initial rotation as described below. The email can
+    be disconsidered. Further rotations can also follow the steps described in
```
Comment (kprosise) [inline source/reference-manual/security/offline-keys.rst:41]:
```suggestion
    be ignored. Further rotations can also follow the steps described in
```

File: source/porting-guide/pg-spl-kernel.rst [hunk 3]
```diff
@@ -84,3 +84,45 @@ Start from the BSP kernel fragment from a close reference board, copy
 over those files relevant to the reference board, change the file name,
 and review the content configurations while reviewing the schematics or
 a list of needed configurations.
+
+Adding a New Kernel Driver
+--------------------------
+
+A simple way to add a new driver or module to the Linux kernel source code is by
```
Comment (kprosise) [inline source/porting-guide/pg-spl-kernel.rst:91]:
```suggestion
The recommended way to add a new driver or module to the Linux kernel source code is by
```

**Review by kprosise — APPROVED** [review]
Minor changes suggested, otherwise it LGTM

**Comment by vanmaegima** [comment]
@kprosise this covers some open items we had on jira. I've applied your suggestions.

**Comment by doanac** [comment]
Docs for 120361fac55b1e108b987d14cb0727d1a36d93c3 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1695/docs/artifacts/html/index.html

File: source/porting-guide/pg-spl-kernel.rst [hunk 4]
```diff
@@ -84,3 +84,46 @@ Start from the BSP kernel fragment from a close reference board, copy
 over those files relevant to the reference board, change the file name,
 and review the content configurations while reviewing the schematics or
 a list of needed configurations.
+
+Adding a New Kernel Driver
+--------------------------
+
+The recommended way to add a new driver or module to the Linux kernel source
```
Comment (angolini) [inline source/porting-guide/pg-spl-kernel.rst:91]: I agree it's a "new kernel driver", but I think it's important to say also that it's an out-of-tree kernel driver recipe that is being added

**Review by angolini — APPROVED** [review]
Only an minor suggestion

**Comment by kprosise** [comment]
After the remaining conversation is resolved, we can go ahead and merge.

**Comment by kprosise** [comment]
@vanmaegima, are we ready to merge? It looks like we are still waiting on another review, but all seems good on my end.

**Comment by vanmaegima** [comment]
@kprosise let's go ahead and merge, thanks for the reminder

**Pull request merged**